packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1067,6 +1067,18 @@ export const MissingIndexForInternationalization = {
 	hint: (src: string) => `Create an index page (\`index.astro, index.md, etc.\`) in \`${src}\`.`,
 } satisfies ErrorData;
 
+/**
+ * @docs
+ * @description
+ * Some internationalization functions can't be exposed unless the default routing is disabled
```

Comment:
Additional context in both the error messages would be great, at least some links in the `@see` thing (see other errors for examples)